trakt/movies.go

```diff
@@ -569,11 +569,12 @@ func PausedMovies(isUpdateNeeded bool) ([]*PausedMovie, error) {
 }
 
 // ToListItem ...
-func (movie *Movie) ToListItem(tmdbMovie *tmdb.Movie) (item *xbmc.ListItem) {
+func (movie *Movie) ToListItem() (item *xbmc.ListItem) {
 	defer perf.ScopeTimer()()
 
-	if tmdbMovie == nil && movie.IDs.TMDB != 0 {
-		if tmdbMovie = tmdb.GetMovie(movie.IDs.TMDB, config.Get().Language); tmdbMovie != nil {
+	var tmdbMovie *tmdb.Movie
+	if movie.IDs.TMDB != 0 {
+		if tmdbMovie := tmdb.GetMovie(movie.IDs.TMDB, config.Get().Language); tmdbMovie != nil {
```

Comment:
You define `tmdbMovie` here locally, so it will not land into this definition: `var tmdbMovie *tmdb.Movie`

Reply:
oh. i missed that. thanks! fixed.